addons/mail/static/src/components/composer_suggested_partners/composer_suggested_partners.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.chatter.ComposerSuggestedPartners" owl="1">
+        <!-- List of followers -->
+        <div class="o_composer_suggested_partners">
+            <t t-foreach='suggestedPartners' t-as='recipient'>
+                <div t-attf-title="Add as recipient and follower (reason: {{recipient.reason}})">
+                    <div class="custom-control custom-checkbox">
+                        <input type="checkbox" class="custom-control-input" t-attf-id="o_composer_suggested_partners_p{{recipient_index}}" t-att-checked="recipient.checked ? 'checked' : undefined" t-att-data-fullname="recipient.full_name"/>
+                        <label t-attf-for="o_composer_suggested_partners_p{{recipient_index}}" class="custom-control-label">
+                            <t t-esc="recipient.name"/>
+                            <t t-if="recipient.email_address">(<t t-esc="recipient.email_address"/>)</t>
+                        </label>
+                    </div>
+                </div>
+            </t>
+            <t t-if='suggestedPartners.length > 3'>
+                <a href="#" class='o_suggested_show_more_less o_suggested_toggle'>
+                    Show <t t-esc="widget.suggestedPartners.length - 3"/> more
```

Comment:
This will be very hard to translate, prefer a getter in js with a string with %s

Note: we'll need to make another pass on this whole part of the template, need to adapt class name to new guideline, and the old logic cannot directly translated.

I know the PR is WIP so I don't go too deep into this now.